spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
> Do you really want to make raw required, even if only for directives?

It's not just for directives, and I do know that parsers are not the only source - that's why I wrote about adding it by tools.

> It's true, AFAICT directive is not required if we have both Expression#parenthesized and Literal#raw.

Yup, that's what I'm thinking about as more proper solution that wouldn't break any of our rules.